x-pack/plugins/index_management/public/application/sections/home/index_list/index_actions_context_menu/index_actions_context_menu.without_redux.tsx

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FunctionComponent } from 'react';
+import { HttpSetup } from '@kbn/core-http-browser';
+import { EuiButtonProps } from '@elastic/eui/src/components/button/button';
+import { EuiPopoverProps } from '@elastic/eui/src/components/popover/popover';
+import { Index } from '../../../../../../common';
+import { reloadIndices } from '../../../../services';
+// @ts-ignore this component needs to be refactored into TS
+import { IndexActionsContextMenu } from './index_actions_context_menu';
+
+export interface ReduxProps {
```

Comment:
This component is a wrapper to initialize the context menu with index actions without using redux. For now all functions are empty and they will be implemented in a follow up PR.